lib/tasks/db.rake

```diff
@@ -0,0 +1,46 @@
+namespace :db do
+
+  desc 'Dumps the database to file'
+  task :dump => :environment do
+
+    host = ActiveRecord::Base.connection_config[:host]
+    db = ActiveRecord::Base.connection_config[:database]
+
+    cmd = "pg_dump --no-owner --no-acl -v -c -f #{dump_file} --no-password "
+    cmd += ENV['DATABASE_URL'] || "--host #{host} --dbname #{db}"
```

Comment:
Inferring from `database.yml`, my thinking is that in CF environments, `DATABASE_URL` must be present or the app won't start. The fallback to `--host ...` --dbname` is used when we are in development i.e. a local db where this should be sufficient. I might be wrong? I should probably add a comment to the code explaining this.